webapp/src/components/ParcelModal/ParcelModal.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, Grid, Header } from 'semantic-ui-react'
+import ParcelPreview from 'components/ParcelPreview'
+import { t } from 'modules/translation/utils'
+import './ParcelModal.css'
+
+export default class ParcelModal extends React.PureComponent {
+  static propTypes = {
+    x: PropTypes.number.isRequired,
+    y: PropTypes.number.isRequired,
+    isDisabled: PropTypes.bool,
+    hasCustomFooter: PropTypes.bool,
+    cancelLabel: PropTypes.string,
+    confirmLabel: PropTypes.string,
+    children: PropTypes.node
+  }
+
+  static defaultProps = {
+    isDisabled: false,
+    hasCustomFooter: false,
+    onCancel: () => {},
+    onConfirm: () => {}
+  }
+
+  render() {
+    const {
+      x,
+      y,
+      title,
+      subtitle,
+      isDisabled,
+      hasCustomFooter,
+      cancelLabel,
+      confirmLabel,
+      onCancel,
+      onConfirm,
+      children
+    } = this.props
+
+    return (
+      <div className="ParcelModal">
+        <div className="modal-column">
+          <div className="modal-preview">
+            <ParcelPreview x={x} y={y} />
+          </div>
+        </div>
+        <div className="modal-column">
+          <div>
+            <Header as="h4" size="large" className="modal-title">
+              {title}
+            </Header>
+            <span className="modal-subtitle">{subtitle}</span>
+          </div>
+          <br />
+          {children ? (
+            <React.Fragment>
+              <div className="modal-children">{children}</div>
+              <br />
+            </React.Fragment>
+          ) : null}
+          {hasCustomFooter ? null : (
```

Comment:
Just a suggestion, when reading the prop from the implementation on other components, I thought the custom footer was the children prop. Might be just me, but maybe this prop could be `hasFooterButtons` or something like that?